Library/Homebrew/os/linux/elf.rb

```diff
@@ -189,6 +197,24 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def save_using_patchelf(new_interpreter, new_rpath)
+    patchelf = DevelopmentTools.locate "patchelf"
+    odie "Could not locate patchelf, please: brew install patchelf." if patchelf.nil?
+    Homebrew.safe_system(
+      patchelf,
+      "--set-interpreter", new_interpreter,
```

Comment:
Handle the case when `new_interpreter` is `nil`.

Reply:
Be sure to test this function `save_using_patchelf` on both ELF executables and ELF libraries.